src/cca/scripts/background.ts

```diff
@@ -82,3 +120,19 @@ Promise.all([serversListPagePromise, intents.GetGlobalIntentInterceptor()]).then
     }
   }
 );
+
+// Add some mock servers if we are running as a Chrome extension.
+serversListPagePromise.then((serverListPage) => {
```

Comment:
Interesting idea. That would also let me mock out unreachable servers which is a limitation of the current approach. Let me get back to you on this one.